### PR TITLE
Add 3 tests

### PR DESCRIPTION
**Pull request by szepeviktor**
```
$ composer test

> find src/ -type f -name '*.php' -print0|xargs -0 -L 1 -- php -l
No syntax errors detected in src/Dismiss.php
No syntax errors detected in src/Notice.php
No syntax errors detected in src/Notices.php
> phpcs --standard=WordPress-Core --exclude=Generic.Arrays.DisallowShortArraySyntax src/
> phpstan analyze
Note: Using configuration file /home/viktor/src/admin-notices/phpstan.neon.dist.
 3/3 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


 [OK] No errors

```

@aristath

### CODE REVIEW

File: src/Dismiss.php [hunk 1]
```diff
@@ -52,6 +52,7 @@ class Dismiss {
 	 * @since 1.0
 	 * @param string $id     A unique ID for this notice. Can contain lowercase characters and underscores.
 	 * @param string $prefix The prefix that will be used for the option/user-meta.
+	 * @param string $scope
```
Comment (aristath) [inline src/Dismiss.php]: Thank you for adding these. Could you add a description for this var as well? The param can be one of `user` or `global` and controls where the dismissal will be saved.

**Review by aristath — CHANGES_REQUESTED** [review]

File: src/Dismiss.php (hunk 1 above)
Comment (szepeviktor) [reply src/Dismiss.php]: Done.

Comment (aristath) [reply src/Dismiss.php]: Thank! I'll go ahead and merge this one :)

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**